### PR TITLE
Basic tests and Travis-CI integration

### PR DESCRIPTION
**Pull request by lox**
This project is woefully short on tests and CI. This PR will add some basic tests and CI via Travis. Feels @joho @mtibben @pda?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,9 @@ FLAGS=-X main.Version=$(VERSION) -s -w
 CERT="Developer ID Application: 99designs Inc (NRM9HVJ62Z)"
 SRC=$(shell find . -name '*.go')
 
+test:
+	go test $(shell go list ./... | grep -v /vendor/)
```
Comment (joho) [inline Makefile]: There's a glide command you can run here instead of using shell

Comment (joho) [inline Makefile]: `go test $(shell glide nv)`

Comment (lox) [inline Makefile]: Indeed, but then it would depends on glide :)

Comment (lox) [inline Makefile]: Which would mean we'd need to install glide on Travis. Because it has vendors committed glide is just needed for updating.

Comment (joho) [inline Makefile]: ah, of course, i saw the use of glide and just assumed from there. travis != our buildkite setup

**Pull request merged**